clustermesh/clustermesh.go

```diff
@@ -26,7 +27,9 @@ import (
 
 	"github.com/blang/semver/v4"
 	"github.com/cilium/cilium/api/v1/models"
+	ciliumK8s "github.com/cilium/cilium/pkg/k8s"
 	ciliumv2 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2"
+	slim_corev1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/api/core/v1"
```

Comment:
Is there any particular reason to use the slim types? They're useful for storing lots and lots and lots of objects in memory, but we're not doing that here, right?

Reply:
I actually don't know. IIRC @sayboras suggested it (but it was months ago, so my memory may fail me here).

Reply:
I don't think there is a reason to use the slim types. Users might actually be interested in fields only provided by the corev1 type. Otherwise they can still filter them out e.g. using `jq`.

I've reverted to use `corev1`. As Casey mentioned, memory usage is not a concern here (which is the primary reason to use the slim types).